service-v2.yaml

```diff
@@ -0,0 +1,15 @@
+name: service-hello
+cluster_env: default_cluster_env_2.4.0_py310:1
+ray_serve_config:
+  import_path: serve_hello:entrypoint
+  runtime_env:
+    working_dir: https://github.com/anyscale/docs_examples/archive/refs/heads/main.zip
+    env_vars:
+      SERVE_RESPONSE_MESSAGE: service v2 was rolled out
+
+# You can optionally specify more fields
+# compute_config: todo_insert_compute_config_name 
+# project: todo_insert_my_project_name
+# version: todo_insert_version
+#
```

Comment:
I think this might just be confusing because it's non-comprehensive and we aren't explaining them. Maybe just omit this for quickstart and we should have explicit docs for what's supported?

Reply:
I think we should remove the optional fields from here and maybe create a more comprehensive sample config that shows how you can set all these other fields in the yaml